test/common/ops/releaseBoth.js

```diff
@@ -78,6 +78,24 @@ describe('shared.ops.releaseBoth', () => {
     expect(user.items.currentMount).to.be.empty;
   });
 
+  it('leaves quest pets and mounts equipped', () => {
```

Comment:
technically it's `non-drop pets` since special pets and premium pets also stay equipped.